glotaran/analysis/scheme.py

```diff
@@ -24,6 +24,7 @@ def __init__(self,
                  model: 'glotaran.model.Model' = None,
                  parameter: ParameterGroup = None,
                  data: typing.Dict[str, typing.Union[xr.DataArray, xr.Dataset]] = None,
+                 extra: typing.Dict[str, xr.DataArray] = None,
```

Comment:
A `file_reader` MUST return a `DataArray`. The requirement for extra is that it can be loaded from disk with a `file_reader`. I could rename it to `extra_data`. Everything else is decided by the model, I don't care what they represent.